sr_handeye_calibration/launch/result_check.launch

```diff
@@ -3,5 +3,6 @@
         <arg name="sim" value="false"/>
     </include>
 
-    <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0.0216801881196 -0.0767846625209 -0.12729356701 0.997677213484 -0.00244016024351 0.00767335748323 0.0676412810249 ra_ee_link camera_link 100" />
+    <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="-0.0829561613453 -0.0502088124026 -0.158993987036 -0.993627195375 0.0450614597081 0.000557050564859 0.103315783666 ra_ee_link camera_link 100" />
```

Comment:
Not sure what this launchfile is for. But will this tf work well for xtion and for deimos?

Reply:
This is to visualise the result of where the hand-eye calibration has determined the camera to be.
To use it, you copy and paste the result from the hand-eye calibration then run it.

Reply:
OK.